utils/utils.go

```diff
@@ -154,6 +154,32 @@ func CreateResultFromEndpoint(ep *api.WorkloadEndpoint) (*current.Result, error)
 	return result, nil
 }
 
+// SanitizeMesosLabel converts a string from a valid mesos label to a valid Calico label.
+// Mesos labels have no restriction outside of being unicode.
+func SanitizeMesosLabel(s string) string {
```

Comment:
I think we need some more unit-level testing of this function, e.g. for labels like the following:

- `-some-label`
- `$mylabel..`
- `//something/here/`
- `some_label-with.lots*of^weird#characters`